src/common.c

```diff
@@ -493,6 +500,39 @@ call_external(gchar** argv)
     return is_successful;
 }
 
+gboolean
+call_external_fork(gchar** argv)
+{
+    // Fork / exec external executable
+    pid_t pid = fork();
+    if (pid == 0) {
+        int x = execvp(argv[0], argv);
+        if (x == -1) {
+            gchar* cmd = g_strjoinv(" ", argv);
```

Comment:
Possibly you could use auto_char here, but looks good nonetheless

Reply:
auto_char where?
What would be the difference with auto_char?

Reply:
If you use auto_char the compiler will take care of free()'ing that newly allocated string.
`            auto_char char* cmd = g_strjoinv(" ", argv);`

Reply:
Thank you. Didn't know that was possible in C.

Reply:
> Thank you. Didn't know that was possible in C.

Yeah, it's possible thanks to a gcc extension.

Please use `auto_gchar` here, see #1819 for details.